app/managers.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+from models import Actor
+import sqlite3
```

Comment:
Still not correct according to PEP8

https://peps.python.org/pep-0008/#imports